ert/ensemble_evaluator/builder/_legacy.py

```diff
@@ -188,19 +208,13 @@ async def _evaluate_inner():
 
             else:
                 logger.debug("ensemble finished normally")
-                result = CloudEvent(
-                    {
-                        "type": identifiers.EVTYPE_ENSEMBLE_STOPPED,
-                        "source": f"/ert/ee/{self._ee_id}/ensemble",
-                        "id": str(uuid.uuid1()),
-                    }
-                )
 
             finally:
                 await timeout_queue.put(None)  # signal to exit timer
                 await send_timeout_future
 
                 # Dispatch final result from evaluator - FAILED, CANCEL or STOPPED
+                assert self._config  # mypy
```

Comment:
Question @sondreso : ValueError or assertion here?

Reply:
I think this is a ValueError